grails-app/i18n/messages_da.properties

```diff
@@ -1,26 +1,155 @@
-default.doesnt.match.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] overholder ikke m�nsteret [{3}]
-default.invalid.url.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] er ikke en gyldig URL
-default.invalid.creditCard.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] er ikke et gyldigt kreditkortnummer
-default.invalid.email.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] er ikke en gyldig e-mail adresse
-default.invalid.range.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] ligger ikke inden for intervallet fra  [{3}] til [{4}]
-default.invalid.size.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] ligger ikke inden for st�rrelsen fra [{3}] til [{4}]
-default.invalid.max.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] overstiger den maksimale v�rdi [{3}]
-default.invalid.min.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] er under den minimale v�rdi [{3}]
-default.invalid.max.size.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] overstiger den maksimale st�rrelse p� [{3}]
-default.invalid.min.size.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] er under den minimale st�rrelse p� [{3}]
-default.invalid.validator.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] overholder ikke den brugerdefinerede validering
-default.not.inlist.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] findes ikke i listen [{3}]
-default.blank.message=Feltet [{0}] i klassen [{1}] kan ikke v�re tom
-default.not.equal.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] m� ikke v�re [{3}]
-default.null.message=Feltet [{0}] i klassen [{1}] kan ikke v�re null
-default.not.unique.message=Feltet [{0}] i klassen [{1}] som har v�rdien [{2}] skal v�re unik
+
```

Comment:
Not familiar with how crowdin manages these files, but just want to confirm that these large empty line sections aren't indicative of an issue

Reply:
I contacted crowdin the past week, because of this and a related issue, they answer me:

> The reason is behind Skip untranslated strings option export option you use in a project. By default, not translated strings are exported with the source text, but Skip untranslated strings just skips that strings and they are exported as empty. Your file has no translations, that is why it has only comments once exported

So with this option unchecked, we get all languages with the English string if they are not translated, with this option checked (as now) we only get empty files if the language is not translated, but, comments and empty lines in the source are downloaded in the translation files. They commented about this:

> At the moment the logic does not allow to skip the files/languages that are not translated yet. However, this improvement is on our roadmap and I will keep you updated on this one!

I cleaned files with only comments and empty lines via some scripting (see "for" script in `crowdin.yml`) but others with some few translations, get the empty lines and comments from source, like this one. See the empty lines in source, for instance:
https://github.com/AtlasOfLivingAustralia/collectory-plugin/blob/b781c701cb021c6302d81b63b6b13b8ff1e283e6/grails-app/i18n/messages.properties#L23

I hope is clear.

If you like I can do some clean of empty lines in this in other related PRs.

Reply:
The empty lines issue maybe is more clear in the files changed in https://github.com/AtlasOfLivingAustralia/bie-plugin/pull/173

Reply:
I personally don't mind the empty lines, particularly in managed files. Was just checking that you knew what the reasoning was, and it sounds like it has a solid reason in this case. No need from me to clean the files up.